deploy/cluster_role.yaml

```diff
@@ -174,6 +174,9 @@ rules:
   - create
   - delete
   - get
+  - list
+  - update
+  - watch
```

Comment:
still don't see the need for the watch permission, is there a way to remove?

Reply:
We are not directly use it. It is used for cache update I think.